test/external/external_test_metrics.py

```diff
@@ -10,7 +10,7 @@ class ExternalTestMetrics(unittest.TestCase):
   def _test_metrics(self, tinygrad_metrics, orig_metrics, pred, label):
     tinygrad_metrics_res = tinygrad_metrics(Tensor(pred), Tensor(label)).squeeze().numpy()
     orig_metrics_res = orig_metrics(torch.from_numpy(pred), torch.from_numpy(label)).numpy()
-    np.testing.assert_equal(tinygrad_metrics_res, orig_metrics_res)
+    np.testing.assert_allclose(tinygrad_metrics_res, orig_metrics_res, atol=1e-6, rtol=1e-3)
```

Comment:
that's very big of an margin, do you need this big?

Reply:
Nop, reduced to atol=1e-8, rtol=1e-7